### PR TITLE
minor corrections and HTC10 device added

### PR DESCRIPTION
**Pull request by pintaf**

### CODE REVIEW

File: README.md
Comment (utzcoz) [inline README.md]: A new PR for this modification has been merged. Maybe you should rebase to the newest code base.

File: default.xml
Comment (utzcoz) [inline default.xml]: Nit: could you move it to the top of maruos part?

Comment (pintaf) [reply default.xml]: I could but that's not really a maruos thing. You would hit the same problem and require this repo while building classic LOS.

Where do you suggest to put it ?
above lxc or above vendor/maruos ?

File: default.xml
Comment (utzcoz) [inline default.xml]: Maybe you should push those projects to maruos. Hi @pdsouza , could you help to grant permission to @pintaf ?

**Review by utzcoz — CHANGES_REQUESTED** [review]

File: default.xml
Comment (utzcoz) [reply default.xml]: Maybe.

**Comment by pintaf** [comment]
@utzcoz PR updated, the HTC10 repo will be updated later once I get access to maruos, or you can integrate them yourself based on my repos.

File: default.xml
Comment (pdsouza) [reply default.xml]: Yes, I will add @pintaf to our team.

**Comment by pdsouza** [comment]
Hi @pintaf, I've just invited you to join our org so you can create your HTC10 repos. Let me know if you have any issues.

File: default.xml
Comment (pdsouza) [reply default.xml]: Hmmm, it's not a huge deal of course but it would be nice if `default.xml` was purely Maru repos. 

Here are some options:

1. The only other file we have where we specify customizations is in `snippets/overrides.xml`. At the bottom of that file we remove some unnecessary LOS repos for example. So it's possible it could go in there but I guess this isn't really an "override".
2. We could possibly create another file in the `snippets` directory called `extras.xml` or something and add it in there.
3. We can just move it to either above or below all the `maruos` repos in `default.xml`.

Any opinions which would be best?

Comment (utzcoz) [reply default.xml]: I prefer to choice 3, because we can only maintain one file for manifest. And the next is choice 2, we can include it with `include` tag.

Comment (pdsouza) [reply default.xml]: I prefer choice 3 as well. @pintaf, can you move this above all the `maruos` repos in our manifest?

Comment (pintaf) [reply default.xml]: I also think choice 3 is the best. However instead of above, I think bellow the maruos repos would be better.
Indeed, this repo need came with my HTC10, but it is also needed by some other devices. Hence it is mostly device related and should be closer between maru repos and devices repos in my opinion.

I'll integrate my HTC repos in maruos and update this PR. Maybe tomorrow, maybe Thursday, I'm sinking under the things to do.

Comment (pdsouza) [reply default.xml]: OK below is fine with me especially since it is device-related. Take your time on the PR, no rush.

**Comment by pintaf** [comment]
Updated, added Pixels repo as well

File: default.xml
Comment (pdsouza) [inline default.xml]: We don't usually include the vendor blob repos in our manifest and usually let devs select whichever method they want to use of obtaining these (including stripping them from factory images with either android-generate-vendor or android-prepare-vendor).

But having the vendor blobs forked in and set up by default in our manifest may make our community's development efforts easier since devices just build out of the box. I don't mind trying this approach for maru-0.7.

Any thoughts, @utzcoz?

**Review by pdsouza — COMMENTED** [review]
Overall looks good to me. Just waiting on @utzcoz's approval before merging.

File: default.xml
Comment (pintaf) [reply default.xml]: if I may, advanced devs know how to get blobs from other sources if they need to.
Including them eases things for people just getting started in building/porting as it was my case.

Maru clearly needs support and committed people. Making things as easy as possible is a way to attract / keep devs.

Comment (utzcoz) [reply default.xml]: Vendor proprietary blob maybe incur `DMCA` from GitHub. What about we update the porting document with the method to get and add those files to specific devices, and add document to our Q&A part to devs?

**Comment by utzcoz** [comment]
> Overall looks good to me. Just waiting on @utzcoz's approval before merging.

This weekend I will test @pintaf 's awesome work for porting work for sailfish and marlin with my devices.

File: default.xml
Comment (pintaf) [reply default.xml]: If DMCA is a problem and I understand it can be, maybe we should not host the blobs in the maru org, but still include links to some proprietary repo so that devices can build out of the box. For example https://github.com/TheMuppets/
This way if some company has to complain against it's blobs being hosted they will ask github to close TheMuppets repo.
So no risks, but clearly improved developer experience.

**Comment by pintaf** [comment]
> > Overall looks good to me. Just waiting on @utzcoz's approval before merging.
> 
> This weekend I will test @pintaf 's awesome work for porting work for sailfish and marlin with my devices.

I don't have the devices with me, so it was a "blind" port. I hope it wil boot

File: default.xml
Comment (pdsouza) [reply default.xml]: I can see arguments both ways. Yes, the dev experience is easier since devices will build out of the box, but on the downside devs will not know the process for obtaining vendor blobs themselves, which is important knowledge if you want to know what's actually happening under the hood.

My main issue with including links to those specific vendor repos in our official manifest may be seen as endorsing them, which we do not do in any way. They are useful to get going quickly, but the files really should be pulled from a trusted factory image or running device.

Given that LineageOS itself does not ship these blob files in their official manifest, I think it's reasonable to assume that Maru OS devs should learn how to obtain these on their own. As @utzcoz suggests, we can update our developer docs with very simple and clear instructions on how to get these blobs, and in the device-specific guide (e.g. [Pixel](https://maruos.com/docs/devices/sailfish.html#install-maru-via-twrp)) we can give explicit instructions on how to obtain the blobs for that device. Also, for the Google devices like the Pixel, we usually use [android-prepare-vendor](https://github.com/anestisb/android-prepare-vendor) to grab these blobs from the official factory image, which is a trusted source.

Sorry @pintaf, but let's take the conservative approach for now. Let's remove the forked blob repos from our org, and remove the manifest links. I'm happy to merge everything else. We can discuss this further later too, but let's at least merge the rest of the PR while we figure out how best to deal with this.

Comment (pintaf) [reply default.xml]: Ok @pdsouza I removed the vendor blobs. Ill try to delete the associated repos from maru org-
Would be nice if you can merge that I have some time to work on this tonight.

**Review by pdsouza — APPROVED** [review]
LGTM. Thanks @pintaf for the quick update!

**Pull request merged**
